src/index.html

```diff
@@ -11,7 +11,163 @@
+      method="post"
+    >
+      <fieldset class="form">
+        <legend>Personal information</legend>
+
+        <label class="form-field">
+          Surname:
+          <input
+            type="text"
+            name="Surname"
+            id="usersSurname"
+          >
+        </label>
+
+        <label class="form-field">
+          Name:
+          <input
+            type="text"
+            name="usersName"
+          >
+        </label>
+
+        <label class="form-field">
+          How old are You?
+          <input
+            type="number"
+            name="usersAge"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+
+        <label class="form-field">
+          Full date of birth:
+          <input
+            type="date"
+            name="dateOfBirth"
+          >
+        </label>
+
+        <label class="form-field">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="terms&conditions"
+            required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="form">
+        <legend>Registration</legend>
+
+        <label class="form-field">
+          E-mail:
+          <input
+            type="email"
+            name="usersEmail"
+            placeholder="email@example.com"
+            required
+          >
+        </label>
+
+        <label class="form-field">
+          Password:
+          <input
+            type="password"
+            name="usersPassword"
+            required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="form">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="form-field">
+          Do you love cats?
+          <label>
+            <input
+            type="radio"
+            name="loveCats"
+            >Yes
+          </label>
+          <label>
+            <input
+            type="radio"
+            name="loveCats"
+            >No
+          </label>
+        </div>
+
+        <label class="form-field">
+          What is your favorite color?
+          <input
+            type="color"
+            name="usersColor"
+          >
+        </label>
+
+        <label class="form-field">
+          What time do you go to bed?
+          <input
+            type="time"
+            name="sleepingTime"
+          >
+        </label>
+
+        <label class="form-field" >
+          What are your favorite brand of cars?
+          <select name="Car brand" multiple>
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </label>
+
+        <label class="form-field">
+          How do you rate our work?
+          <input
+            type="range"
+            name="range"
+            minlength="1"
+            maxlength="10"
+          >
```

Comment:
Use `min` and `max` attributes instead of `minlength` and `maxlength` for the range input. For example, `min="1" max="10"` 🎚️